app/routes/items/partial_move.js

```diff
@@ -0,0 +1,8 @@
+import AuthorizeRoute from './../authorize';
+
+export default AuthorizeRoute.extend({
+  model(params){
+     var item = this.store.peekRecord("item", params.item_id);
+     return item || this.store.findRecord('item', params.item_id);
```

Comment:
Do we need to declare item as var? May be we can directly do this
`return this.store.peekRecord("item", params.item_id) || this.store.findRecord('item', params.item_id)`